sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/zookeeper/ZooKeeperClient.java

```diff
@@ -171,7 +171,12 @@ private Map<String, List<String>> fuzzyGetConfigListByGroupAndKey(String key, St
             if (!child.contains(nodeName)) {
                 continue;
             }
-            String childPath = parentNodePath + child;
+            String childPath;
+            if (parentNodePath.endsWith(String.valueOf(ZK_PATH_SEPARATOR))) {
+                childPath = parentNodePath + child;
+            } else {
+                childPath = parentNodePath + ZK_PATH_SEPARATOR + child;
+            }
```

Comment:
just childPath = parentNodePath + ZK_PATH_SEPARATOR + child  maybe ok

Reply:
parentNodePath may be /, can't just do childPath = parentNodePath + ZK_PATH_SEPARATOR + child